request.schema.json

```diff
@@ -116,6 +120,16 @@
                 "capabilities",
                 "renderers"
             ]
+        },
+        {
+            "required": [
+                "request_uuid",
+                "language",
+                "timestamp",
+                "highChartsData",
```

Comment:
I'd include the context field here as well. It may be useful for stuff like a surrounding caption/title. But not really blocking.